_posts/2023-07-31-paris-dev-sprint.md

```diff
@@ -0,0 +1,93 @@
+    email: francois.goupil@inria.fr
+    website: https://github.com/francoisgoupil
+    image: "francois_goupil.jpeg"
+---
+<div>
+  {% include postauthor.html %}
+</div>
+
+
+In June of 2023 (week of June 19 to 23), the scikit-learn team held its first developers sprint since 2019! The sprint took place in Paris, France at the Dataiku office. The sprint event was an in-person event only and had 29 participants.
+
+The following [scikit-learn team members](https://scikit-learn.org/stable/about.html) joined the sprint:  
+
+1. Gael Varoquaux (@GaelVaroquaux)
+1. Olivier Grisel (@ogrisel)
+1. Guillaume Lemaitre (@glemaitre)
+1. Tim Head (@betatim)
+1. Maren Westermann
+1. Arturo Amor Quiroz
+1. Adrin Jalali
+1. Loïc Estève
+1. Thomas Fan
+1. Julien Jerphanion (@jjerphan)
+1. François Goupil (@francoisgoupil)
+1. Roman Yurchak
+1. Joris Van den Bossche
+
+The following community members joined the sprint:  
+
+1. Ines (intern at Dataiku)
+1. Chaine San Buenaventura
+1. Yann Lechelle
+1. Vincent Maladière
+1. Franck Charras
+1. Jovan Stojanovic
+1. Leo Dreyfus-Schmidt
+1. Alexandre Landeau
+1. Louis Fouquet
+1. Alexandre Vigny
+1. Harizo Rajaona
+1. Samuel Ronsin
+1. Riccardo Cappuzzo
+1. Camille Troillard
+1. Denis Engemann
+
+
+<figure>
+ <img src="/assets/images/posts_images/2023-paris-sprint/paris_2023.jpg" alt="group of people who participated in the sprint" max-width="20%" max-height="20%" /> 
+ <figcaption>
+ scikit-learn Developer Sprint, Paris, June 2023; Photo credit: <a href=" "> Inria / Photo B. Fourrier, June 2023</a>; (from left to right, back to front): XXX, XXX, François Goupil, Tim Head, Guillaume Lemaitre, XXX, XXX, Gael Varoquaux, Olivier Grisel, Alexandre Gramfort [add other people in]
+ </figcaption>
+</figure>
+
+## Sponsors
+- Dataiku provided the space and some of the food, as well as all of the coffee.
+- The scikit-learn consortium paid for the lunch, the travel and accommodation expenses.
+
+## Topics covered at the sprint
+- PR #13649: [Monotonic constraints for Tree-based models](https://github.com/scikit-learn/scikit-learn/pull/13649) 
+- Discussed the vision/future directions for the project. What is important to keep the project relevant in the future.
+- Should we share some points beyond the vision statement?
+- Thomas F will try and create a vision statement
+- Discussed what people are keeping an eye on with a two year time scale in mind in terms of technology and developments that are relevant.
+- Tim: keep improving our documentation (not just expanding it but also “gardening” to keep it readable)
+- Tim: increase active outreach and communication about new features/improvements and other changes. A lot of cool things in scikit-learn are virtually unknown to the wider public (e.g. Hist grad boosting being on par with lightgbm in terms of performance, …)
+
+
+### What is next?
+
+We are discussing co-locating with OpenML next Sep to Dec 2024 in Berlin to organize another developers' sprint. 
```

Comment:
Next spring

Reply:
changing to "2024" in general.  
"spring" is different for different hemispheres, so will keep it general.